contributors/contributors.json

```diff
@@ -1759,11 +1759,6 @@
     "AvatarUrl": "https://avatars.githubusercontent.com/u/21339406?v=4",
     "HtmlUrl": "https://github.com/tom-dudley"
   },
-  {
-    "Name": "tomaszbartoszewski",
-    "AvatarUrl": "https://avatars.githubusercontent.com/u/7828978?v=4",
-    "HtmlUrl": "https://github.com/tomaszbartoszewski"
-  },

```

Comment:
@Jericho Any idea why this contributor is being removed? The user doesn't seem to have been renamed

Reply:
The reason AddinDiscoverer is removing this contributor from the json and yaml files is that GitHub no longer returns his name when we invoke the API endpoint to get the list of contributors. So the question becomes: why is GitHub no longer returning his name? I don't know the answer to that; you would have to ask GitHub.

As I understand it, GitHub's definition of "contributor" is someone who has committed to a repository. Is it possible that this person's commit was deleted? Did he contribute a PR which was previously accepted but subsequently deleted? Was the repo he contributed to deleted or moved out of cake-build (maybe a repo was moved from cake-build to cake-contrib?). These are just guesses on my part to try to explain why someone would no longer be deemed by GitHub to be a contributor of a given repo.

The only other guess I can come up with is that GitHub returns a maximum number of names when we invoke the API to get the contributors (I was under the impression the max was 500, but I could be wrong). Maybe we have reached this limit and for some reason his name is no longer one of the first 500? This is a far fetched guess because I don't think we have reached the 500 mark on any of the 20 repos under cake-build, but I thought I would mention it anyway.

Reply:
Continuing investigation: I see this person contributed to `cake-build/website` back in October 2018
![image](https://user-images.githubusercontent.com/112710/229148864-52c7f1d8-2394-46ba-9a63-6dc582e2d184.png)

Both PRs have been merged, the commits are still in the repo, the website repo was not moved to cake-contrib. Also I checked and there are 135 contributors to the website repo so it seems very unlikely this person's would be omitted because we have reached GitHub's max number of contributors returned in a single API call.

All this to say that I am at a loss to explain why GitHub is no longer returning his name. Like I said before, you'll need to contact GitHub to get an explanation.

Reply:
From GitHub's [documentation](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-a-projects-contributors#troubleshooting-contributors):

![image](https://user-images.githubusercontent.com/112710/229154225-3300c115-2e41-4827-baae-f97668f9da97.png)

The first bullet point about being in the "top 100" explains why this person does not show up [here](https://github.com/cake-build/website/graphs/contributors) but, as far as I understand, this "top 100" limit does not apply when we fetch the contributors via the API. So this does not explain why his name is excluded when we invoke the API endpoint.

The third bullet about "email address used to author the commits" might be a possible explanation. Could it be that this person recently changed the email address on their GitHub profile and this caused GitHub to no longer be able to associated him with the commits from October 2018?

Reply:
@augustoproiete just a reminder that this opened PR is blocking AddinDisco from further synchronizing the list of contributors. Are you still trying to figure out why GitHub is no longer returning this person's name when we query the GitHub API for the list of contributors?

Reply:
@Jericho Apologies for the delay. I'll merge this one to unblock AddinDisco, and will continue the investigation when I have a chance.